composer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "10up/jobber",
+  "description": "Jobber WordPress Plugin.",
+  "license": "MIT",
+  "type": "wordpress-plugin",
+  "authors": [
+    {
+      "name": "10up",
+      "email": "info@10up.com"
+    }
+  ],
+  "require": {
+    "php": ">=8.3",
```

Comment:
Do we want to be this aggressive on the PHP version? We typically still use 7.4 as our minimum for our OSS plugins